tectonic/dpx-cs_type2.c

```diff
@@ -731,7 +731,7 @@ do_charstring (card8 **dest, card8 *limit,
       do_operator2(dest, limit, data, endptr);
     } else if (b0 < 32 && b0 != 28) { /* 19, 20 need mask */
       do_operator1(dest, limit, data, endptr);
-    } else if ((b0 <= 22 && b0 >= 27) || b0 == 31) { /* reserved */
+    } else if ((b0 >= 22 && b0 <= 27) || b0 == 31) { /* reserved */
```

Comment:
I'm not sure where the "real" (x-)dvipdfm(x) upstream is, but I'd also suggest pointing out this fix to them.